db/migrations/20221108152353_create-charge-versions.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+const tableName = 'charge_versions'
+
+exports.up = async function (knex) {
+  await knex
+    .schema
+    .withSchema('water')
+    .createTable(tableName, table => {
+      // Primary Key
+      table.uuid('charge_version_id').primary().defaultTo(knex.raw('gen_random_uuid()'))
+
+      // Data
+      table.string('licence_ref')
+      table.string('scheme')
+
+      // Automatic timestamps
+      table.timestamps(false, true)
+    })
+
+  await knex.raw(`
+    CREATE TRIGGER update_timestamp
+    BEFORE UPDATE
+    ON water.${tableName}
+    FOR EACH ROW
+    EXECUTE PROCEDURE update_timestamp();
+  `)
+}
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
```

Comment:
I think we can drop this documentation. IMHO it's noise we don't need in the migrations. While we at it, we should add `use strict` to the top of the file.

```suggestion
```